Makefile

```diff
@@ -980,6 +981,12 @@ publish: image
 	docker push $(DOCKER_IMAGE):$(VERSION)
 	if [ -f e/Makefile ]; then $(MAKE) -C e publish; fi
 
+.PHONY: publish-ecr
+publish-ecr: image
+	docker tag $(DOCKER_IMAGE) $(DOCKER_IMAGE_ECR)
+	docker push $(DOCKER_IMAGE_ECR):$(VERSION)
+	if [ -f e/Makefile ]; then $(MAKE) _C e publish-ecr; fi
```

Comment:
```suggestion
	if [ -f e/Makefile ]; then $(MAKE) -C e publish-ecr; fi
```

Reply:
Fixed in https://github.com/gravitational/teleport/pull/14347/commits/70756d0696cb76b8862fd029d5f02ed419b827b9